.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: android
+before_install:
+- openssl aes-256-cbc -K $encrypted_0b5eff37ea82_key -iv $encrypted_0b5eff37ea82_iv
+  -in secrets.tar.enc -out secrets.tar -d
+- tar xvf secrets.tar
+- cp $TRAVIS_BUILD_DIR/my-release-key.keystore $TRAVIS_BUILD_DIR/android/app/my-release-key.keystore
+- cp $TRAVIS_BUILD_DIR/keystore.properties $TRAVIS_BUILD_DIR/android/keystore.properties
+- nvm install 6
+- node --version
+install:
+- npm install
+android:
+  components:
+  - tools
+  - build-tools-23.0.1
+  - build-tools-25.0.3
+  - platform-tools
+  - android-23
+  - extra-android-m2repository
+  - extra-google-google_play_services
+  - extra-google-m2repository
+  - addon-google_apis-google-16
+  licenses:
+  - android-sdk-preview-license-.+
+  - android-sdk-license-.+
+  - intel-android-extra-license.+
+script:
+- cd android && ./gradlew assembleRelease
+deploy:
+  provider: releases
+  api_key:
+    secure: okpBUibQ+A+4CDvgUlMIq8kw1w9MfmjKLe2RHCAFfwVFHlCKgdDaw2rCXgkP4wcpBkcQqH0WiP0/PqeF/o+Y8K5Co6GRbaPBnCkIFQXWM8fqrICFK0fPY3K2dXUTgYXbsM5bp8P2IC38trhuWNKiOlXCsQL7Q/DVqUdpvc3pmqYAliavOWQHj8xzhnq8otzIjknWiqt7f1f7p68kAD33j8ZoCpdwz/mIAhIYcKT3VibNscJYBFxvdbCIKyH392ai5bXk3ThQVeKAEQtnqW93tM7fXfYOHLQdrKpsGz02mgw0E64i/ulN4wSmNEpWnPqED+WaxoBfYc3cAylvLh0Eom32BZmnSvJTTKSHJYNtxAXbSxvgAAZz4qAcjHccDfu2wQuJ3j1rfaGjtG4kro62tG8jpIHgc6VAPlBXNF9rsfMn5XHaBvTe/mChOD+EY6bvBix+24V5i09CbXCZwxf+cpWqdse5nkaL8GmFqmWX0bLAQHj6DGy1Z1MCTIeZTbpVu5ocbFeOXHwPEHbm2Fb55qyx8YmvPvCOtqRE7nqe5JYPWpPraMJUfg0s32Vo8/kBzU6Hkm5hPmOMSPQDrNdMzin+4Zej1/nkWNdAdFgkXua8sG4C3e4ltx2YxWFNAfrhxlc+9R04Ja8RUns8m/0zbQXThEmZhVkxYufb0xq/tq8=
```

Comment:
are u sure sure travis.yml should be inside the repo??
is there any sensible data? (api_key,...)?
 
if that is the case, you can forget this PR and revoke all sensible data.

Reply:
just to be sure

Reply:
1- it has to be,
2- all secured keys or critic data is encrypted, no worries (cf travis documentation).
3- it all just fine! 👍

Reply:
![image](https://user-images.githubusercontent.com/1345675/29883951-22ab9816-8db2-11e7-974b-e9113a445626.png)